torchvision/prototype/transforms/_utils.py

```diff
@@ -18,10 +18,12 @@
 
 # Type shortcuts:
 DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]
-FillType = Union[int, float, Sequence[int], Sequence[float]]
+ImageType = Union[torch.Tensor, PIL.Image.Image, features.Image]
+TensorImageType = Union[torch.Tensor, features.Image]
+FillType = Union[int, float, Sequence[int], Sequence[float], None]
```

Comment:
Including `None` here ensures it works both when we provide directly a `None` value and when the value is in a dictionary.